types/ineum/ineum-tests.ts

```diff
@@ -0,0 +1,35 @@
+/// <reference types="./index" />
```

Comment:
I think you can remove this line.

Reply:
Hi @minestarks, I included it because the `index.ts` is not exporting a module, therefore the testfile did not find the types via the `tsconfig.json`.

When removing the line, the testfile would be marked as full of errors.

Reply:
Strange, I can remove that line and the file compiles fine and the tests pass locally with `npm run lint ineum`. How are you running the tests?

Reply:
You are right. I checked it again. Must have been an error from mid-development where the error just temporarily occurred. I remove the type reference.